src/browser/components/preferences/main-inc-xhtml.patch

```diff
@@ -1,8 +1,21 @@
 diff --git a/browser/components/preferences/main.inc.xhtml b/browser/components/preferences/main.inc.xhtml
-index ff577c035ff6128f7ae948bc810082c92b3a624c..40f94fff53c63bdff28eaa68f9b0736d53c5b4f8 100644
+index ff577c035ff6128f7ae948bc810082c92b3a624c..eda462e81c448ddd3d63726d94caa6702500ab73 100644
 --- a/browser/components/preferences/main.inc.xhtml
 +++ b/browser/components/preferences/main.inc.xhtml
-@@ -141,7 +141,8 @@
+@@ -74,6 +74,12 @@
+     <checkbox id="ctrlTabRecentlyUsedOrder" data-l10n-id="ctrl-tab-recently-used-order"
+               preference="browser.ctrlTab.sortByRecentlyUsed"/>
+ 
++    <checkbox id="ctrlTabCycleByAttribute" data-l10n-id="ctrl-tab-cycle-by-attribute"
```

Comment:
I'd move this into zen's "tab management" settings section, and avoid making patches